src/ziggurat/mapper.clj

```diff
@@ -49,13 +54,17 @@
                 end-time    (.toEpochMilli (Instant/now))]
             (metrics/report-time (str metric-namespace ".execution-time") (- end-time start-time))
             (case return-code
-              :success (metrics/increment-count message-processing-namespace "success")
-              :retry (do (metrics/increment-count message-processing-namespace "retry")
-                         (producer/retry-for-channel message topic-entity channel))
-              :skip (metrics/increment-count message-processing-namespace "skip")
-              :block 'TODO
+              :success (do (metrics/increment-count message-processing-namespace "success" topic-entity-name)
+                           (metrics/increment-count "message-processing" "success" topic-entity-name))
+              :retry   (do (metrics/increment-count message-processing-namespace "retry" topic-entity-name)
+                           (metrics/increment-count "message-processing" "retry" topic-entity-name)
+                           (producer/retry-for-channel message topic-entity channel))
+              :skip    (do (metrics/increment-count message-processing-namespace "skip" topic-entity-name)
+                           (metrics/increment-count "message-processing" "skip" topic-entity-name))
```

Comment:
Extract this magic string to let binding

Reply:
done